flux/src/actions/index.tsx

```diff
@@ -3,6 +3,66 @@ import { KubeObject } from '@kinvolk/headlamp-plugin/lib/lib/k8s/cluster';
 import { useSnackbar } from 'notistack';
 import React from 'react';
 
+function ForceReconciliationAction(props) {
+  const { enqueueSnackbar } = useSnackbar();
+  const [open, setOpen] = React.useState<boolean>(false);
+  const { resource } = props;
+  if(!resource.jsonData.spec.force) {
+    return null;
+  }
+  const forceReconciliationType = resource.jsonData.spec.force ? 'Disable' : 'Enable';
+
+  return (
+    <>
+      <ActionButton
+        description={`${forceReconciliationType} force reconciliation`}
+        icon={resource.jsonData.spec.force ? 'mdi:stopwatch-check' : 'mdi:stopwatch-cancel-outline'}
+        onClick={() => {
+          setOpen(true);
+        }}
+      />
+      <ConfirmDialog
+        open={open}
+        handleClose={() => setOpen(false)}
+        onConfirm={() => {
+          setOpen(false);
+          resource.jsonData.spec['force'] = !resource.jsonData.spec.force;
+          const patch = resource.constructor.apiEndpoint.patch;
+          patch(
+            {
+              spec: {
+                force: false,
+              },
+            },
+            resource.jsonData.metadata.namespace,
+            resource.jsonData.metadata.name
+          )
+            .then(response => {
+              if (!response.spec.force) {
+                enqueueSnackbar(
+                  `Successfully ${forceReconciliationType}d force reconciliation for ${resource.metadata.name}`,
```

Comment:
As a general rule, never generate words for human readable messages like this. Just use `Disabled force reconciliation for ${resource.metadata.name}` and `Enabled force reconciliation for ${resource.metadata.name}`